lage.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  pipeline: {
+    ['build-tools']: ['^build-tools'],
```

Comment:
Is the `['attribute']: [val]` syntax necessary - can it not be like the others?

Reply:
Because this is just a .js file, dashes are not allowed in the keys of objects unless they are declared in string form. They work fine, but have to be declared in this manner.